index.html

```diff
@@ -148,29 +148,28 @@ <h2 class="pt-3" id="info">What <em>is</em> Standard?</h2>
           href="https://magic.wizards.com/en/content/standard-formats-magic-gathering" rel="noopener">
             <b>Standard</b>
           </a>
-          is a rotating group of <i>Magic: The Gathering</i> sets. Most sets enter Standard when they're released and
-          drop out about twenty-one months later.
+          is a tournament format which consists of several recent <i>Magic: The Gathering</i> sets. Most sets enter
+          the format when they're released and drop out about twenty-one months later.<br>
+          The rotating group of sets generally grows until it contains eight. Then—with the next joining release—the oldest four sets
+          are dropped at once with the four most recent ones remaining. Hence the fresh one is added as fifth legal set before the pool
+          grows up to eight again. There are exceptions to this. It's confusing.
           </p>
           <p>
-          Generally Standard will grow until it would receive its ninth set, then instead it drops its oldest four sets
-          to receive it as a fifth set. There are exceptions to this. It's confusing.
-          </p>
-          <p>
-          A <b>Standard card</b> is a card printed or reprinted into a set currently in Standard.
+          A <b>Standard card</b> is a card in a set currently part of the legal pool.
           <a v-tippy class="tip"
-             title="e.g. As long as Naturalize has a reprint currently in Standard, all previous prints of Naturalize
-             are also considered Standard.">
+             title="Sets not only contain newly designed cards. Sometimes there are so called reprints of former cards as well.<br>
+                    e.g. As long as Naturalize has a reprint currently in Standard, all previous prints of Naturalize
+                    are also considered Standard legal.">
```

Comment:
This is a bit lengthy for how simple the concept is, and similar to above the `<br>` could stand to be replaced with `<p>` for more separation, or removed if the shorter length doesn't call for separation anymore.
```suggestion
             title="Different prints of a card count as the same card.
                    e.g. As long as Naturalize has a reprint that's Standard-legal, all previous prints of Naturalize
                    are also Standard-legal.">
```

Reply:
Do you want to use "Standard-legal" two times in the same sentence here?

Reply:
I think it can be any of "in Standard", "Standard-legal", or "legal"; but yeah I do think it's important to repeat the same exact thing twice to make the point clear: "If `thing A` has `quality X`, `thing B` also has `quality X`."

IMO saying it in two different ways makes the sentence harder to parse.

Reply:
Oh, I didn't want to say it in two different ways just to not reuse the word or to avoid another occurrence of Standard by all means. I would reference the first again.

Something along the lines of `... e.g. As long as Naturalize has a reprint that's Standard-legal, all previous prints of Naturalize are as well.`